.travis.yml

```diff
@@ -60,9 +60,9 @@ jobs:
         - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.3.2
         - export PATH="$HOME/.yarn/bin:$PATH"
       install:
-        - cd web/app && yarn && yarn webpack
+        - ./bin/web build
```

Comment:
As far as I can tell, the CI failure happening on this branch stems from the fact that we're no longer running `yarn` without any arguments -- we're just running `yarn webpack`, which is failing. I think you can fix this with:

```
      install:
        - ./bin/web setup
        - ./bin/web build
```